e2e/src/config/jest/global-setup.ts

```diff
@@ -16,10 +16,12 @@ export default async (): Promise<void> => {
     deployContract(new DummyContract__factory(), account),
     deployContract(new DummyContract__factory(), l2Account),
   ]);
-
   console.log(`L1 Dummy contract deployed at address: ${await dummyContract.getAddress()}`);
   console.log(`L2 Dummy contract deployed at address: ${await l2DummyContract.getAddress()}`);
 
+  const l2TestContract = await deployContract(new TestContract__factory(), l2Account);
+  console.log(`L2 Test contract deployed at address: ${await l2TestContract.getAddress()}`);
```

Comment:
What is the impact of having this deploy here? 

This seems to be for the txExclusion API, so IMO it shall be there and not add delay to the remaining testsuite.

Reply:
I'd prefer to leave it here in the global-setup.ts, because:
- I tried to put them in the beforeAll() of txExclusion test suite but noticed it was not a good idea as it's no longer guaranteed to run before all tests in the test suite when `log.concurrent` is used (jest [issue](https://github.com/jestjs/jest/issues/4281) here)
- when put in `global-setup.ts`, it will only deploy once for the whole e2e test (i.e. all contracts will be deployed before it starts running the test suites in parallel), so we didn't deploy more than it shouldn't

Reply:
Btw, to play safe, I removed all `beforeAll` in all e2e test suites where i`t.concurrent` is used due to the [jest](https://github.com/jestjs/jest/issues/4281) issue mentioned above